package.json

```diff
@@ -59,7 +59,8 @@
         "request": "^2.88.0",
         "rss-parser": "^3.7.3",
         "rxjs": "^6.6.7",
-        "symbol-sdk": "^1.0.2-alpha-202108061451",
+        "symbol-sdk": "^1.0.3",
+        "symbol-statistics-service-typescript-fetch-client": "1.1.3-alpha-202111191644",
```

Comment:
Let's please drop alpha tags across the organization. CC @AnthonyLaw. See my comment here: https://github.com/symbol/symbol-bootstrap/pull/333#issuecomment-980807106

Reply:
Agreed, we need to release `version 1.1.3` for `symbol-statistics-service-typescript-fetch-client` first. I'm on it.